yktracec/src/jitmodbuilder.cc

```diff
@@ -1806,10 +1806,12 @@ class JITModBuilder {
               VMap[CI] = ConstantPointerNull::get(
                   Type::getInt8PtrTy(JITMod->getContext()));
 
-              // The next two trace blocks are the (unmappable) control point
-              // followed by the return to this block which only has a branch
-              // instruction left which we don't care about. So we can just skip
-              // over these and continue with the block after.
+              // The next two trace blocks may be skpped as they only contain
```

Comment:
s/skpped/skipped/

Reply:
@ptersilie I think just this needs fixing?

Reply:
[061d09c](https://github.com/ykjit/yk/pull/703/commits/061d09c6490da35edce50ae22725020fb094a9a5)